lava-slave/Dockerfile

```diff
@@ -1,5 +1,7 @@
 FROM baylibre/lava-slave-base:2018.11-1_bpo9_1
 
+RUN apt-get update
```

Comment:
Doesn't this need `DEBIAN_FRONTEND=noninteractive` and `-y` ?

Reply:
No since this command is not interactive